### DISCUSSION TITLE
Does the applications that integrate with onlyoffice community server must use AGPL?

### DISCUSSION THREAD

**Discussion by qianwch** [ONLYOFFICE/DocumentServer#1871] [Q&A]
I meant integrate with, not embed in, use api to launch up onlyoffice.

**Comment by ShockwaveNN** [comment]
> onlyoffice community server

Hi, do you mean [ONLYOFFICE CommunityServer](https://github.com/ONLYOFFICE/CommunityServer/) product or ONLYOFFICE DocumentServer community (opensource) edition?

**Reply by qianwch** [reply]
Sorry for the late response, i meant the ONLYOFFICE DocumentServer community (opensource) edition.

**Reply by ShockwaveNN** [reply]
Ok, thanks, I'll show you question to our legal team

**Reply by qianwch** [reply]
Just like there are onlyoffice integrating plugins for jira or confluence, but both jira and confluence are not open source product. Could our application integrate with onlyoffice documentserver community in the same way as them?
https://github.com/ONLYOFFICE/onlyoffice-jira
